packages/drupal/silverback_autosave/silverback_autosave.libraries.yml

```diff
@@ -0,0 +1,16 @@
+drupal.silverback_autosave:
+  version: VERSION
```

Comment:
As @Leksat mentioned on Slack, we can drop this line

Reply:
The point is correct, but instead of dropping I prefer to define the version (and this have full control over CDN in future updates)